### PR TITLE
Node.js Tutorials Cheat Sheet: Updated references

### PR DESCRIPTION
**Pull request by TiagoDanin**
## Description of new Instant Answer, or changes
Updated references.

## People to notify
@moollaza 

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/testing-reference/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/nodejs_tutorials_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/nodejs-tutorials.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
             },
             {
                 "link": "http://nodeguide.com/",
-                "key": "Node.js Guide"
+                "key": "Node.js guide"
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/nodejs-tutorials.json]: Why not capitalize this when all the others are.

File: share/goodie/cheat_sheets/json/nodejs-tutorials.json [hunk 2]
```diff
@@ -27,16 +27,16 @@
     "sections": {
         "Tutorials": [
             {
-                "link": "http://nodeschool.io/",
-                "key": "NodeSchool.io"
+                "link": "https://stackoverflow.com/documentation/node.js/340/getting-started-with-node-js",
+                "key": "Getting started with Node.js"
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/nodejs-tutorials.json]: I'm ok with this. The SO answer is more direct than the node school site.

File: share/goodie/cheat_sheets/json/nodejs-tutorials.json [hunk 3]
```diff
@@ -117,24 +137,32 @@
             {
                 "link": "http://www.pluralsight.com/search/?searchTerm=Node.js",
                 "key": "Pluralsight courses (paid)"
+            },
+            {
+                "link": "https://www.google.co.in/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwj6oMvR35nTAhWFuY8KHfC-BFoQFggZMAA&url=https%3A%2F%2Fwww.udemy.com%2Funderstand-nodejs%2F&usg=AFQjCNF3WkS8cFyYrTgzgd5V_S1ANKkZJw&sig2=rD4t8JYDPvQKjwAMzQnH7Q",
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/nodejs-tutorials.json]: Why the google.co.in link here, this must have been a mistake.

File: share/goodie/cheat_sheets/json/nodejs-tutorials.json [hunk 4]
```diff
@@ -117,24 +137,32 @@
...
+            {
+                "link": "https://www.youtube.com/watch?v=-u-j7uqU7sI&list=PL6gx4Cwl9DGBMdkKFn3HasZnnAqVjzHn_",
+                "key": "The New Boston"
             }
         ],
         "Screencasts": [
             {
                 "link": "http://learnallthenodes.com/",
-                "key": "Learn All the Nodes"
+                "key": "Learn All The Nodes"
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/nodejs-tutorials.json]: Lower case for the similar to 'a' or 'in' when in title.

File: share/goodie/cheat_sheets/json/nodejs-tutorials.json [hunk 5]
```diff
@@ -168,7 +196,7 @@
             },
             {
                 "link": "http://jsbooks.revolunet.com/",
-                "key": "A detailed list of free JavaScript Books"
+                "key": "Most detailed list of free JavaScript Books"
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/nodejs-tutorials.json]: I don't like this change. Most detailed list according to who? I'd remove 'Most' and capitalize detailed.

File: share/goodie/cheat_sheets/json/nodejs-tutorials.json [hunk 6]
```diff
@@ -208,7 +236,7 @@
             },
             {
                 "link": "http://howtonode.org/",
-                "key": "How to Node"
+                "key": "How To Node"
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/nodejs-tutorials.json]: Lower case to like in or a.

**Comment by mintsoft** [comment]
@sahildua2305 Good for merge?

File: share/goodie/cheat_sheets/json/nodejs-tutorials.json [hunk 7]
```diff
@@ -150,7 +150,7 @@
         "Screencasts": [
             {
                 "link": "http://learnallthenodes.com/",
-                "key": "Learn All The Nodes"
+                "key": "Learn All the Nodes"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/nodejs-tutorials.json]: Nit: Why is "All" capitalized?

**Pull request merged**